.travis.yml

```diff
@@ -1,20 +1,27 @@
 sudo: false
+
 language: node_js
+
 addons:
   sauce_connect: true
+
 cache:
   directories:
     - node_modules
+
 node_js:
-  - 0.1
-  - 4.3
+  - node
   - stable
```

Comment:
This is not quite right.

 - `node` is synonymous with `stable`
 - We want to keep testing LTS versions (while 4.3 is outdated, we should update this to 4.5 or maybe just 4)
 - We still want to test Node 0.1, until we stop and that will be a breaking change